detection_rules/integrations.py

```diff
@@ -74,8 +74,8 @@ def find_least_compatible_version(package: str, integration: str,
 
     def compare_versions(int_ver: str, pkg_ver: str) -> bool:
         """Compares integration and package version"""
-        pkg_major, pkg_minor = Version(pkg_ver)
-        integration_major, integration_minor = Version(int_ver)[:2]
+        pkg_major, pkg_minor, _ = Version(pkg_ver)
```

Comment:
roll back changes

Reply:
The comparison for this is to confirm major's are equal, so `compare_versions` as a function was removed to simplify this code.